sw/device/lib/BUILD

```diff
@@ -23,12 +23,11 @@ cc_library(
     name = "pinmux",
     srcs = ["pinmux.c"],
     hdrs = ["pinmux.h"],
-    deprecation = "Deprecated pre-DIF code. We prefer you use the DIFs.",
```

Comment:
SG, yeah I think thats a good place for it. We currently don't have a good mechanism for organizing/versioning the test utils, or any SW really, according to the top-level it corresponds to, but since the `pinmux_init` applies to both top-levels currently, it doesn't sound like that will be an issue?

Reply:
Yeah, we're okay for now. And I got my nomenclature wrong--It does call multiple DIFs. :)

Also, done.